Source/Particles/PhysicalParticleContainer.cpp

```diff
@@ -1114,6 +1160,32 @@ PhysicalParticleContainer::AddPlasma (int lev, RealBox part_realbox)
                     p_optical_depth_BW[ip] = breit_wheeler_get_opt(engine);
                 }
 #endif
+                // Initialize user-defined integers with user-defined parser
+                for (int ia = 0; ia < n_user_int_attribs; ++ia) {
+#if defined(WARPX_DIM_3D) || defined(WARPX_DIM_RZ)
+                    pa_user_int_data[ia][ip] = static_cast<int>(user_int_parserexec_data[ia](pos.x, pos.y, pos.z, u.x, u.y, u.z, t));
```

Comment:
Thinking about it: actually we `pos.x` and `.y` should be well defined (to zero) in all our code paths. We can simplify this and the section below to just use the general line.

Reply:
good point! thank you!